austation/code/modules/research/designs/bluespace_designs.dm

```diff
@@ -0,0 +1,12 @@
+/////////////////////////////////////////
+//////////////Bluespace//////////////////
+/////////////////////////////////////////
+/datum/design/bluespace_belt
+	name = "Belt of Holding"
+	desc = "An astonishingly complex belt popularized by a rich blue-space technology magnate."
+	id = "bluespace_belt"
+	req_tech = list("bluespace" = 7, "materials" = 5, "engineering" = 5, "plasmatech" = 6)
```

Comment:
This is an incompatible research system. Paradise doesn't use techwebs. Please add belts to a techweb node and change code accordingly